downstream/modules/platform/ref-single-eda-controller-with-internal-db.adoc

```diff
@@ -0,0 +1,68 @@
+
+[id="ref-single-eda-controller-with-internal-db_{context}"]
+
+= Single {EDAcontroller} node with internal database
+
+Use this example to populate the inventory file to install {EDAcontroller}. This installation inventory file includes a single {EDAcontroller} node with an internal database.
+
+[IMPORTANT]
+====
+{ControllerNameStart} must be installed before you populate the inventory file with the following {EDAName} variables.
+====
+
+-----
+# Automation EDA Controller Configuration
+#
+
+automationedacontroller_admin_password='<eda-password>'
+
+automationedacontroller_pg_host=''
+automationedacontroller_pg_port=5432
+
+automationedacontroller_pg_database='automationedacontroller'
+automationedacontroller_pg_username='automationedacontroller'
+automationedacontroller_pg_password='<password>'
+
+# The full routable URL used by EDA to connect to a controller host.
+# This URL is required if there is no Automation Controller configured
+# in inventory.
+#
+automation_controller_main_url = 'https://controller.example.org/'
+' 
```

Comment:
Not sure if the single quote on this new line is a typo or if its part of the adoc formatting.

Reply:
Thanks for the alert! Will definitely remove it. It could have been a simple typo or something that occurs in asciidoctor (typing the ` doesn't produce just one mark, it automatically displays two of them, I'm assuming due to required syntax for some attributes).